### PR TITLE
[enhancement](metric)Add metric for exec_state prepare function

### PR DESCRIPTION
**Pull request by ByteYue**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Previously, we found out that sometimes the rpc service might timeout due to the extremely time consuming operation during exec_state prepare, thus this pr adds one metric to monitor the prepare time cost of each single fragment.
After compiling with this commit, BE would be able to generate data compatible with grafana, and you can watch the data using dashboard like below.
image
As for the total time consumption of all the fragments prepare, you can refer to the brpc port to watch the rpc metric for more detailed message.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [x] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/admin-manual/maint-monitor/monitor-metrics/metrics.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ curl http://be_host:webserver_port/metrics?type=json
 || `{status="skip",type="report_all_tablets"}` | Num | xxx 类型任务被跳过执行的次数的累计值 | |
 |`doris_be_fragment_endpoint_count`| | Num| 同 | FIXME: 同 `doris_be_data_stream_receiver_count` 数目。并且向量化引擎缺失 |
 |`doris_be_fragment_request_duration_us`| | 微秒| 所有 fragment intance 的执行时间累计 | 通过斜率观测 instance 的执行耗时 | P0 |
+|`doris_be_fragment_prepare_duration_ns`| | 纳秒| 单个 fragment intance 的prepare时间累计 | 通过斜率观测 instance 的prepare耗时 | P0 |
```
Comment (morningman) [inline docs/zh-CN/docs/admin-manual/maint-monitor/monitor-metrics/metrics.md:213]: How about use same unit as `doris_be_fragment_request_duration_us`?

**Review by morningman — DISMISSED** [review]
LGTM,
and maybe we can move all metrics to bvar in future.

**Comment by dataroaring** [comment]
please update the figure above.

File: be/src/runtime/fragment_mgr.cpp [hunk 2]
```diff
@@ -61,6 +63,7 @@ namespace doris {
 DEFINE_GAUGE_METRIC_PROTOTYPE_2ARG(plan_fragment_count, MetricUnit::NOUNIT);
 DEFINE_GAUGE_METRIC_PROTOTYPE_2ARG(timeout_canceled_fragment_count, MetricUnit::NOUNIT);
 DEFINE_GAUGE_METRIC_PROTOTYPE_2ARG(fragment_thread_pool_queue_size, MetricUnit::NOUNIT);
+bvar::LatencyRecorder g_fragmentmgr_prepare_latency("doris_FragmentMgr", "prepare");
```
Comment (dataroaring) [inline be/src/runtime/fragment_mgr.cpp:66]: In my memory, latency of bvar should be used with window.

**Review by dataroaring — CHANGES_REQUESTED** [review]

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 38.53 seconds
 load time: 558 seconds
 storage size: 17154699358 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221028043958_clickbench_pr_35107.html

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 39.13 seconds
 load time: 572 seconds
 storage size: 17154644815 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221028115203_clickbench_pr_35384.html

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**